### DISCUSSION TITLE
P-GALLERIA THUMBNAIL AUTO HIDDEN

### DISCUSSION THREAD

**Discussion by nurulischweizer** [primefaces/community#1109] [PrimeNG] [answered]
P-galleria thumbnail auto hidden when hover or select the image. 

<div role="tablist" class="p-galleria-thumbnail-items p-items-hidden" style="">

How to not make this run, i already use css to fixed this but still cannot work. 

:host::ng-deep .gallery .p-galleria .p-galleria-thumbnail-item:hover {
  visibility: visible !important;
}

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Right now only supported option is whether to display them on hover or not. Unfortunately, making thumbnails permanently visible is not an option for now. However, it's a good feature idea and we'll wait for the community votes.
